app/views/DefaultBrowserView/Item.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { memo } from 'react';
+
+import I18n from '../../i18n';
+import { TSupportedThemes } from '../../theme';
+import { themes } from '../../lib/constants';
+import * as List from '../../containers/List';
+import { IBrowsersValues, TValue } from '.';
+
+interface IRenderItem extends IBrowsersValues {
+	browser: string | null;
+	theme: TSupportedThemes;
+	changeDefaultBrowser: (newBrowser: TValue) => void;
+}
+
+const Item = memo(({ title, value, browser, changeDefaultBrowser, theme }: IRenderItem) => {
```

Comment:
Is this really necessary?
Considering the amount of options available, I don't see the need to use memo in this component.

It's not a matter of being right or wrong, it's a matter of making the simple complex.